runtests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
```

Comment:
The file is missing executable bit.

`chmod +x runtests.sh`

Reply:
@msrb Do we need this, I didn't see this command in worker. :)

Reply:
It's there in worker ;)

```bash
$ ls -la runtest.sh
-rwxrwxr-x. 1 michal michal 3897 Apr 11 20:51 runtest.sh
```

But it's missing here:
```bash
$ ls -la runtests.sh 
-rw-rw-r--. 1 michal michal 694 May  6 17:30 runtests.sh
```

Running `chmod +x runtests.sh` should do the trick here.